cmd/mr_test.go

```diff
@@ -109,7 +109,7 @@ func Test_mrCmd_MR_description_and_options(t *testing.T) {
 		}
 	})
 	t.Run("create MR from file", func(t *testing.T) {
-		git := exec.Command("git", "checkout", "mrtest")
+		git := exec.Command("git", "checkout", "-b", "local/mrtest", "origin/mrtest")
```

Comment:
I'm a bit wary of overloading a the "create MR from file" test this way.  Instead of this I suggest adding a completely new test for this functionality.